vmm/src/memory_manager.rs

```diff
@@ -255,11 +255,13 @@ const LENGTH_OFFSET_HIGH: u64 = 0xC;
 const STATUS_OFFSET: u64 = 0x14;
 const SELECTION_OFFSET: u64 = 0;
 
-// The MMIO address space size is subtracted with the size of a 4k page. This
-// is done on purpose to workaround a Linux bug when the VMM allocates devices
-// at the end of the addressable space.
+// The MMIO address space size is subtracted with the size of a 64k page. This
```

Comment:
Just a nit, but afaik there's no such thing as a 64k page on x86.

Reply:
AFAIR the term is indeed used at some places in the MSDN doc. On Linux it would probably be just 16*4k pages. Fixed, thanks.